### PR TITLE
add grapviz node info

### PR DESCRIPTION
**Pull request by jievince**
add outputVar, inputVar, colNames on node
image

### CODE REVIEW

File: printer/plandescprinter.go [hunk 1]
```diff
@@ -175,10 +198,25 @@ func (p PlanDescPrinter) renderDotGraph() string {
...
+					val := string(pair.GetValue())
+					if key == "inputVar" {
+						inputVar = val
+					} else if key == "colNames" {
+						colNames = graphvizString(val)
+					}
+				}
+			}
+			builder.WriteString(fmt.Sprintf("\t%s [label=\"%s|outputVar: %s\\l|inputVar: %s\\l|colNames: %s\\l\", shape=Mrecord];\n",
+				planNodeName, planNodeName, outputVar, inputVar, colNames))
```
Comment (yixinglu) [inline printer/plandescprinter.go:216]: Please extract to a function for this general node string.

Comment (yixinglu) [reply printer/plandescprinter.go:216]: same as line 102~118

File: printer/plandescprinter.go [hunk 2]
```diff
@@ -60,6 +67,25 @@ func condEdgeLabel(condNode *graph.PlanNodeDescription, doBranch bool) string {
 	return ""
 }
 
+func defaultPlanNode(planNodeDesc *graph.PlanNodeDescription, planNodeName string, builder *strings.Builder) {
```
Comment (yixinglu) [inline printer/plandescprinter.go:70]: It's better to let this function return string rather than pass in StringBuilder.

Comment (jievince) [reply printer/plandescprinter.go:70]: OK

**Review by yixinglu — APPROVED** [review]

**Pull request merged**